docs/cheatsheet.rst

```diff
@@ -114,10 +129,14 @@ The steps are as follows:
   hub issue create
   <enter text>
 
-2. Start a new feature with hubflow named: issue-<issue # you created in step 1>-<description of work>
+2. Start a new feature with hubflow named: :code:`issue-<issue # you created in step 1>-<description of work>`
 ::
   hub hf feature start issue-1-going-to-write-some-code
+
 3. Write code
+
+This is where the actual magic happens.
```

Comment:
<img width="773" alt="cheatsheet_ _praekelt_ways_of_working_1_documentation" src="https://cloud.githubusercontent.com/assets/165585/12443418/5d7adc8a-bf60-11e5-8dfd-02b5391c227e.png">

@hodgestar i think it looks better unindented.